wicket-intro/CafeAddress/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It was generated by the maven command I used to build the scaffold. I'm not using logging in this example, so I will remove it.